esphomeyaml/sensor.csv

```diff
@@ -9,7 +9,7 @@ Dallas, components/sensor/dallas, ds18b20.jpg
 DHT, components/sensor/dht, dht22.jpg
 DHT12, components/sensor/dht12, dht12.jpg
 ESP32 Hall Sensor, components/sensor/esp32_hall, magnet.svg
-HDC1080, components/sensor/hdc1080, HDC1080.jpg
+HDC1080, components/sensor/hdc1080, hdc1080.jpg
```

Comment:
Oh, thanks for the fix 😁 (My laptop is running with a case-insensitive file system so I didn't even notice that :)